src/pages/deriv-go/_banner.js

```diff
@@ -103,8 +103,8 @@ const BannerWrapper = styled(Flex)`
 `
 const ButtonDerivGO = styled(Button)`
     padding: 1.5rem 1.6rem;
-    height: 64px;
-    width: 100%;
+    height: 59px;
```

Comment:
can `height: 60px` work?

Reply:
Yeah I was testing on chrometools the height to match the figma but 60px could work too